gnovm/cmd/gno/testdata/lint/file_error_txtar

```diff
@@ -15,6 +15,9 @@ func TestIHaveSomeError() {
 	fmt.Println("Hello", 42)
 }
 
+-- gno.mod --
+module gno.land/p/test
```

Comment:
If you agree with my comment, I suggest keeping some, if not all, txtar gnomod-less packages as non-draft.

Reply:
the reason I added it is that missing the gno.mod file prints a warning from the linter

because in these tests we're checking for specific kinds of errors, I'd keep the gno.mod files here